scripts/heat/devstack-heat-template.yaml

```diff
@@ -0,0 +1,164 @@
+heat_template_version: 2016-10-14
+
+description: |
+  devstack on Leap 15.0
+
+parameters:
+  key_name:
+    type: string
+    label: Key-pair name
+    description: Name of key-pair to be used for compute instance
+
+  floating_network_name:
+    type: string
+    label: Floating network name
+    description: Name of key-pair to be used for compute instance
+    default: floating
+
+  flavor_name:
+    type: string
+    label: Flavor
+    description: Flavor of the server
+    default: m1.large
+
+  image_name:
+    type: string
+    label: Image
+    description: Image to boot the server with
+    default: openSUSE-Leap-15.0
+
+  run_tempest:
+    type: boolean
+    label: Run Tempest
+    description: Set to 'yes' to run Tempest tests.  Defaults to 'no'.
```

Comment:
set to "true" ?

Reply:
Currently the main (well, only) use case is for manual hacking.  If we start using it in CI then we can easily switch the default.

Reply:
I mean use true/false instead of yes/no

Reply:
Ohhhh haha I see!  Yeah thanks, good catch :-)